python/tests/lang/test_definition.py

```diff
@@ -13,6 +13,11 @@
 
 
 class TestDefinition(TestCase):
+    def test_uuid(self):
+        self.assertTrue(create_schema_definition("Test").uid.is_safe)
+        self.assertEqual(create_schema_definition("Test"), create_schema_definition("Test"))
+        self.assertNotEqual(create_schema_definition("Test1"), create_schema_definition("Test2"))
+
```

Comment:
We could add an additional test in the definitions test to demonstrate that a large list of definitions all have unique uuids. It's not a significantly larger number than the existing test when you think about the small chance for collisions, but 60 definitions does provide a little more peace of mind. This test can go into the language context test if you'd prefer -- I was on the fence about where it should end up.

You'll also need to add the import line
```
from aac.lang.active_context_lifecycle_manager import get_active_context
```

```suggestion

    def test_uuid_in_active_context(self):
        definition_uuids_list = [definition.uid for definition in get_active_context().definitions]
        definition_uuids_set = set(definition_uuids_list)

        self.assertEqual(len(definition_uuids_list), len(definition_uuids_set))
        
```

Reply:
I agree, I'm going to put it in the language context tests since we're specifically accessing the active context - it seems more appropriate there, IMO.